src/components/Checkbox/Checkbox.tsx

```diff
@@ -6,9 +6,28 @@ import { FC, InputHTMLAttributes } from 'react';
 import styles from './Checkbox.module.css';
 
 type Props = {
+  /**
+   * サイズ
+   * @default medium
+   */
   size?: 'medium' | 'small';
-} & Required<Pick<InputHTMLAttributes<HTMLInputElement>, 'value' | 'children'>> &
-  Omit<InputHTMLAttributes<HTMLInputElement>, 'size'>;
+  /**
+   * ネイティブ要素のname属性。グループ化したい項目に同じ名前をつける
+   */
+  name: string;
```

Comment:
I was worried but decided `name` prop was required.
I don't think this is necessary in an example where a stand-alone checkbox would be used.
However, we have determined that there are many cases where `name` prop is forgotten.